src/ucp/core/ucp_worker.c

```diff
@@ -2520,6 +2567,8 @@ ucs_status_t ucp_worker_create(ucp_context_h context,
 
     ucp_worker_create_vfs(context, worker);
 
+    ucp_worker_set_max_am_header(worker);
```

Comment:
i would do it before line 2558
in case vfs would show max am header

Reply:
fixed